packages/gatsby/content/advanced/questions-and-answers.md

```diff
@@ -60,9 +60,26 @@ If you're not using Zero-Installs:
 .pnp.*
 ```
 
+The files we recommend to **commit** to your repository can depend on the type of project you're creating (Y = Yes, N = No, ? = Maybe):
+
+|                          | Libraries | Applications | [Zero-Installs](/features/zero-installs) |
+|--------------------------|-----------|--------------|---------------|
+| `.yarn/install-state.gz` | N         | N            | N             |
+| `.yarn/unplugged`        | N         | N            | Y             |
+| `.yarn/build-state.yml`  | N         | N            | Y             |
+| `.yarn/plugins`          | Y         | Y            | Y             |
+| `.yarn/releases`         | Y         | Y            | Y             |
+| `.yarn/versions`         | Y         | Y            | Y             |
+| `.yarn/sdks`             | ?         | Y            | Y             |
+| `.yarn/cache`            | ?         | Y            | Y             |
+| `.pnp.*`                 | ?         | Y            | Y             |
+| `yarn.lock`              | Y         | Y            | Y             |
+| `.yarnrc.yml`            | Y         | Y            | Y             |
```

Comment:
This section sounds a bit overly verbose considering that right above there's already the ready-to-copy gitignore entries. Additionally, the difference between "libraries and application" is virtually none, since apps don't necessarily need/want sdks/cache/pnp file either.